### DISCUSSION TITLE
Cannot access GUI over network

### DISCUSSION THREAD

**Discussion by eyeseesharp** [koush/scrypted#1403] [Q&A]
I installed Scrypted for the first time via link here https://docs.scrypted.app/installation.html

for Linux + Docker install.

After running it says the server can now be accessed via the url and ending in :10443. If I do that though, the page never loads.

I cannot make out any relevant errors from the log, pasted below.

s6-rc: info: service s6rc-oneshot-runner: starting
s6-rc: info: service s6rc-oneshot-runner successfully started
s6-rc: info: service fix-attrs: starting
s6-rc: info: service setup: starting
SCRYPTED_DOCKER_AVAHI != true, won't manage dbus nor avahi-daemon
s6-rc: info: service setup successfully started
s6-rc: info: service dbus: starting
s6-rc: info: service dbus successfully started
s6-rc: info: service avahi: starting
SCRYPTED_DOCKER_AVAHI != true, not starting dbus-daemon
s6-rc: info: service fix-attrs successfully started
s6-rc: info: service legacy-cont-init: starting
s6-rc: info: service avahi successfully started
SCRYPTED_DOCKER_AVAHI != true, not starting avahi-daemon
s6-rc: info: service legacy-cont-init successfully started
s6-rc: info: service legacy-services: starting
s6-rc: info: service legacy-services successfully started
#######################################################
Scrypted Volume           : /server/volume
Scrypted Server (Local)   : https://localhost:10443/
Scrypted Server (Remote)  : https://172.18.0.1:10443/
Scrypted Server (Remote)  : https://10.0.0.5:10443/
Scrypted Server (Remote)  : https://fe80::42:a5ff:fe99:abcd:10443/
Scrypted Server (Remote)  : https://fe80::456:4bff:6f0:abcd:10443/
Scrypted Server (Remote)  : https://fe80::d407:7eff:fe25:abcd:10443/
Scrypted Server (Remote)  : https://fe80::1436:b5ff:fef6:abcd:10443/
Scrypted Server (Remote)  : https://fe80::4c4d:deff:fe6c:abcd:10443/
Version:       : 0.96.0
#######################################################
Scrypted insecure http service port: 11080
Ports can be changed with environment variables.
https: $SCRYPTED_SECURE_PORT
http : $SCRYPTED_INSECURE_PORT
Certificate can be modified via tls.createSecureContext options in
JSON file located at SCRYPTED_HTTPS_OPTIONS_FILE environment variable:
export SCRYPTED_HTTPS_OPTIONS_FILE=/path/to/options.json
https://nodejs.org/api/tls.html#tlscreatesecurecontextoptions
#######################################################

what else could be at play?

**Comment by bvanpeski** [comment]
Sme issue here. My log looks similar, but can't access the Scrypted site at `10443`.

**Comment by austinrdennis** [comment]
OK, I'm in the same boat but this is the port it's running on for you, OP:

> Scrypted insecure http service port: 11080

**Comment by haokaibo** [comment]
Hi @austinrdennis ,
I got the same issue. And I tried the links below, none of them works. Could you please give me some hints? Thanks.

http://localhost:11080
https://localhost:11080
https://localhost:10443

### What I did

I installed the scrypted on my docker via Portainer. While I cannot visit the web UI from the web browser neither with the https://localhost:10443 or http://localhost:11080. The firefox/chrome web browser just says "Unable to connect" when I visit those links. The state of the scrypted docker container is "running".  And the image I used is koush/scrypted.

<img width="1280" alt="Screenshot 2024-07-06 at 11 25 44 AM" src="https://github.com/koush/scrypted/assets/181500/3512b4f1-a245-4e73-b956-9a25e7216999">

**Reply by haokaibo** [reply]
Hi folks,
Finally, I figured it out by asking the GenAI for installing Scrypted on Mac which running with Apple Silicon Chip. 

> Here is the answer below which fixs my issue.

Ensure you have Docker Desktop for Mac installed and running on your Apple Silicon Mac.
Open Terminal on your Mac.
Pull the Scrypted Docker image. You'll need to ensure you're using an ARM64 compatible image. The command might look like:
```shell
docker pull koush/scrypted:latest
```
(Verify the correct image name and tag from Scrypted's official documentation)
Run the Scrypted container. The command might be similar to:
```shell
docker run -d \
  --name=scrypted \
  --restart=unless-stopped \
  -p 10443:10443 \
  -v ~/.scrypted:/server/volume \
  koush/scrypted:latest
```
Once the container is running, you should be able to access Scrypted by opening a web browser and navigating to:
```shell
https://localhost:10443
```
